src/app/components/game-center/game-center.component.ts

```diff
@@ -1,27 +1,46 @@
+import { ToastrService } from 'ngx-toastr';
 import { Component, OnInit, ViewChild } from '@angular/core';
 import { MatDialog, MAT_DIALOG_DATA } from '@angular/material/dialog';
 import { ErrorRoundDialogComponent } from '../error-round-dialog/error-round-dialog.component';
 import { GuessWordService } from 'src/app/services/guess-word.service';
 
+
 @Component({
   selector: 'app-game-center',
   templateUrl: './game-center.component.html',
   styleUrls: ['./game-center.component.scss'],
 })
 export class GameCenterComponent implements OnInit {
+  constructor(
+    private dialog: MatDialog,
+    private guessWord: GuessWordService,
+    private toastr: ToastrService
+  ) {}
 
-  constructor(private dialog: MatDialog, private guessWord: GuessWordService) {}
-  ngOnInit(){
+  ngOnInit() {
     this.openDialog();
   }
 
   word = '';
-
+  encontrado = false;
```

Comment:
```suggestion
  found = false;
```

variables, functions and stuff, allways in english